mne/io/constants.py

```diff
@@ -799,8 +799,11 @@
 FIFF.FIFFV_COIL_DIPOLE             = 200  # Time-varying dipole definition
 # The coil info contains dipole location (r0) and
 # direction (ex)
-FIFF.FIFFV_COIL_FNIRS_HBO               = 300  # fNIRS oxyhemoglobin
-FIFF.FIFFV_COIL_FNIRS_HBR               = 301  # fNIRS deoxyhemoglobin
+FIFF.FIFFV_COIL_FNIRS_HBO             = 300  # fNIRS oxyhemoglobin
+FIFF.FIFFV_COIL_FNIRS_HBR             = 301  # fNIRS deoxyhemoglobin
+FIFF.FIFFV_COIL_FNIRS_RAW             = 302  # fNIRS raw light intensity
+FIFF.FIFFV_COIL_FNIRS_OD              = 303  # fNIRS optical density
+FIFF.FIFFV_COIL_FNIRS_CHROMA          = 304  # fNIRS chromophore concentration
```

Comment:
@larsoner can you check these and see how to accept this in the fif standard?

Reply:
@rob-luke isn't `FIFFV_COIL_FNIRS_CHROMA` redundant with the existing `FIFFV_COIL_FNIRS_HBO` and `FIFFV_COIL_FNIRS_HBR`?

Do we need / should we have pairs for RAW and OD like we do for chroma (currently HBO/HBR)?

Reply:
Exactly what I have been wondering. 

* For Raw and chroma we cant have pairs because we dont know what wavelength and how many wavelengths we will have. The NIRx machine uses 760 and 850 nm. But other machines use different wavelengths. Depends if we want this solution to be general or just work for this particular machine. We could have `FIFFV_COIL_FNIRS_RAW_WAVELENGTH1`  ... `FIFFV_COIL_FNIRS_RAW_WAVELENGTH10` , but then we would still need to encode the frequency in the channel names.

* I dont know what to do about the existing HbO and HbR types. Because there are other chroma too. Should I add `FIFFV_COIL_FNIRS_HBT` and `FIFFV_COIL_FNIRS_OXCCO` too?

* Having a design where we encode information in the channel names for `raw` and `od`, it then seems odd to have a difference system for `hbo/hbr`

Any suggestions/comments welcomed here, I think this is the last big decision to be made.

Reply:
This makes me think that we should indeed just keep 300/301 HbO/HbR for backward compat, and add 302/303/304. We can add a helper function to convert a file with old coil types (HbO/HbR, 300/301) to the new scheme CHROMA (304) with a channel rename. Then make the picking functions respect both coil types and channel naming (for `fnirs`). It's unfortunate to waste the existing types knowing we won't use them anymore, but it seems like the cleanest way to overcome our initial design choices.

Reply:
Ok final thought on this topic... we could kick the can down the road for a bit regarding adding coil type 304. I need to close this PR, then add a raw to od converter. Then we could discuss this again in when I commit a od to chroma converter (and have had more experience with MNE by then). The downside would be that we would have to request FIFF changes twice.

Reply:
It's not too bad to make multiple requests. In the first one I mention that we'll probably need a couple more soon(ish), then they won't be surprised / annoyed at the increased volume. So I agree let's just add what we need now and punt on the rest for now. So I'll ask about just `FIFFV_COIL_FNIRS_RAW` and `FIFFV_COIL_FNIRS_OD`.